### PR TITLE
Support `increase/rate` function in the `MQE` query language.

### LINKED ISSUE #11500: [Feature] Support rate, increase and abs functions in MQE(query and alerting)
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

These three functions could help the users to  identify the existence of the peak during the given time range.

Rate is for cases like values od traffic, suddenly much more or less traffic could be the beginning of chaos.
Compare to rate, increase(negative or positive) is for a certain value.
Abs is to make expression easier, and avoid greater or less than at both sides

### Use case

The context has been discussed https://github.com/apache/skywalking/discussions/11483

### Related issues

_No response_

### Are you willing to submit a pull request to implement this on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wankai123**
- [ ] If this is non-trivial feature, paste the links/URLs to the design doc.
- [X] Update the documentation to include this new feature.
- [ ] Tests(including UT, IT, E2E) are added to verify the new feature.
- [ ] If it's UI related, attach the screenshots below.

- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #11500.
- [X] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: docs/en/api/metrics-query-expression.md [hunk 1]
```diff
@@ -296,6 +296,37 @@ view_as_seq(not_existing, service_cpm)
...
 
+## Trend Operation
+Trend Operation takes an expression and performs a trend calculation on its results.
+
+Expression:
+```text
+<Trend-Operator>(Metrics Expression, time_range)
+```
+
+`time_range` is the number of the calculates range, should be a positive integer. The unit will automatically adjusts according to the query [Step](../../../oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/enumeration/Step.java),
```
Comment (wu-sheng) [inline docs/en/api/metrics-query-expression.md:307]:
```suggestion
`time_range` is the positive int of the calculated range. The unit will automatically align with to the query [Step](../../../oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/enumeration/Step.java),
```

File: docs/en/api/metrics-query-expression.md [hunk 2]
```diff
@@ -296,6 +296,37 @@ view_as_seq(not_existing, service_cpm)
...
+
+
+| Operator | Definition                                                                            | ExpressionResultType          |
+|----------|---------------------------------------------------------------------------------------|-------------------------------|
+| increase | returns the increase in the time range in the time series                             | TIME_SERIES_VALUES |
+| rate     | returns the per-second average rate of increase in the time range in the time series  | TIME_SERIES_VALUES |
+
+For example:
+If we want to query the increase value of the `service_cpm` metric in 2 minute(assume the query Step is MINUTE),
+we can use the following expression:
```
Comment (wu-sheng) [inline docs/en/api/metrics-query-expression.md:318]: Could you be more clear about this? If `increase in 2 minutes`, it should be T1 - 2, and T2 - 2.
The time sequence should be `T1-2`, `T2-2`, T1, T2.

I am not following your description. Am I getting this wrong?

Comment (wankai123) [reply docs/en/api/metrics-query-expression.md:318]: yes, the time sequence won't change, it's the metrics' time sequence.
Assume the step is minute,  the metrics result is `T1, T2, T3`. Then the `increase in 2 minutes` result is `V(T1)-V(T1-2), V(T2)-V(T2-2), V(T3)-V(T1)`.  
Is this description better？

Comment (wankai123) [reply docs/en/api/metrics-query-expression.md:318]: The description in the doc: `Assume the T3 point, the increase = T3 - T1, If the metric value(T3 or T1) is empty, the result will be empty.`
is described if the value of T1 or T3 is empty, in `Timepoint T3`, the increased value will be empty.

File: docs/en/api/metrics-query-expression.md [hunk 3]
```diff
@@ -296,6 +296,46 @@ view_as_seq(not_existing, service_cpm)
...
+```
+
+`time_range` is the positive int of the calculated range. The unit will automatically align with to the query [Step](../../../oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/enumeration/Step.java),
+for example, if the query Step is `MINUTE`, the unit of `time_range` is `minute`.
+
+
+| Operator | Definition                                                                            | ExpressionResultType          |
+|----------|---------------------------------------------------------------------------------------|-------------------------------|
+| increase | returns the increase in the time range in the time series                             | TIME_SERIES_VALUES |
+| rate     | returns the per-second average rate of increase in the time range in the time series  | TIME_SERIES_VALUES |
```
Comment (wu-sheng) [inline docs/en/api/metrics-query-expression.md:314]: Should we consider supporting labeled value as well?

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**